capabilityreceiver/pkg/capabilityhandler/validation/r4validation.go

```diff
@@ -242,20 +226,26 @@ func checkResourceList(capStat capabilityparser.CapabilityStatement, rule endpoi
 	return ruleError
 }
 
-// SmartHTTPResponseValid checks if the SMART-on-FHIR response code is 200 using the base
-// HTTPResponse function, and then adds specific R4 reference information
-func (v *r4Validation) SmartHTTPResponseValid(smartHTTPRsp int) endpointmanager.Rule {
-	baseComment := "FHIR endpoints requiring authorization SHALL serve a JSON document at the location formed by appending /.well-known/smart-configuration to their base URL."
-	baseRule := v.baseVal.HTTPResponseValid(smartHTTPRsp)
-	baseRule.RuleName = endpointmanager.SmartHTTPRespRule
-	baseRule.Comment = baseComment
-	baseRule.Reference = "http://www.hl7.org/fhir/smart-app-launch/conformance/index.html"
-	baseRule.ImplGuide = "USCore 3.1"
-	if (smartHTTPRsp != 0) && (smartHTTPRsp != 200) {
-		strResp := strconv.Itoa(smartHTTPRsp)
-		baseRule.Comment = "The HTTP response code was " + strResp + " instead of 200. " + baseComment
+// SmartResponseExists checks if the SMART-on-FHIR response exists
+func (v *r4Validation) SmartResponseExists(smartRsp smartparser.SMARTResponse) endpointmanager.Rule {
+	ruleError := endpointmanager.Rule{
+		RuleName:  endpointmanager.SmartRespExistsRule,
+		Valid:     true,
+		Expected:  "true",
+		Actual:    "true",
+		Comment:   "The SMART Response exists.",
```

Comment:
I think we should use `"FHIR endpoints requiring authorization SHALL serve a JSON document at the location formed by appending /.well-known/smart-configuration to their base URL."` from the old SmartHttpResponseValid Rule